rectified: true

numOfFrames: 2358
seq: "00"

imagesPath: "/home/christos/catkin_ws/src/mini_project_kokas/src/vio_slam/images/demo_1/"

fileExtension: ".jpg"

Camera_l:
  fx: 264.4025
  fy: 264.225
  cx: 334.2375
  cy: 188.45625
  k1: 0.0
  k2: 0.0
  p1: 0.0
  p2: 0.0
  k3: 0.0

Camera_l_path: /zed2i/zed_node/left_raw/image_raw_color
# /zed2i/zed_node/left_raw/image_raw_color
# /camera_1/left/image_raw

Camera_r:
  fx: 264.4025
  fy: 264.225
  cx: 334.2375
  cy: 188.45625
  k1: 0.0
  k2: 0.0
  p1: 0.0
  p2: 0.0
  k3: 0.0
  
Camera_r_path: /zed2i/zed_node/right_raw/image_raw_color
# /zed2i/zed_node/right_raw/image_raw_color
# /camera_1/right/image_raw

Camera:
  width: 640
  height: 360
  fps: 15.0
  bl: 0.119342

Camera_path: /zed2i/zed_node/left_raw/image_raw_color
ground_truth_path: /odom/ground_truth/2
pointcloud_path: /camera_1/points2

IMU:
  T_b_c1: 
    rows: 4
    cols: 4
    dt: f
    data: [  0.0040000, -0.0000080,  0.9999920, -0.008,
            -0.9999840,  0.0040000,  0.0040000, 0.023,
            -0.0040000, -0.9999920,  0.0000080, -0.000,
            0.0, 0.0, 0.0, 1.0]
  path: /imu/data

Stereo:
  T_c1_c2:
    rows: 4
    cols: 4
    dt: f
    data: [1.0, 0.0, 0.0, -0.119342,
         0.0, 1.0, 0.0, 0.0,
         0.0, 0.0, 1.0, 0.0,
         0.0, 0.0, 0.0, 1.0]